INSTALL

```diff
@@ -85,3 +88,59 @@ copy and paste these lines into the shell:
 You should see lmi's GUI. Try "File | New | Illustration", then press "OK";
 you should see values. Then try "File | Print preview"; you should see an
 illustration in your 'pdf' viewer.
+
+
+Install using Windows Subsystem for Linux (Windows 10)
+---
+
+(1) Install WSL
+
+Open a "Windows PowerShell" window as Administrator:
```

Comment:
I understand that we need Administrator privileges for installing WSL, but it could be nice to separate the parts that can't be done without them from the parts that can be done as a normal user. This would notably allow installing lmi under WSL without administrator privileges if WSL had been already preinstalled.

Reply:
OK, this actually seems to have been already addressed, with the second invocation not requiring administrative privileges any more, thanks.